ephios/api/templates/oauth2_provider/application_list.html

```diff
@@ -3,18 +3,41 @@
 {% load i18n %}
 {% block settings_content %}
     <div class="block-center">
-        <h3 class="block-center-heading">{% trans "Your applications" %}</h3>
+        <h3 class="block-center-heading">{% trans "OAuth2 applications" %}</h3>
+        <a class="btn btn-primary" href="{% url "oauth2_provider:register" %}">{% trans "New Application" %}</a>
         {% if applications %}
```

Comment:
I was asked what OAuth is and what you can configure here, that isn't obvious to most planners. A short paragraph here about what this page is for would be great

Reply:
its meant for staff users, not planners, but we sure can add a description and call it "3rd party integration" or smth like that?!